lib/combinat.gi

```diff
@@ -2448,10 +2448,12 @@ InstallGlobalFunction( "IteratorOfPartitions", function( n )
 
 #############################################################################
 ##
-#F  IteratorOfPartitionsSet( <set> )
+#F  IteratorOfPartitionsSet( <set> [, <k> [, <flag> ] ]  )
 ##
-##  all unordered partitions of the set <A>set</A> into pairwise disjoint
-##  nonempty sets.
+##  All unordered partitions of the set <A>set</A> into pairwise disjoint
+##  nonempty sets. If <A>k</A> is given, then all unordered partitions of
+##  size <A>k</A> of the set <A>set</A> are given. If additonally <A>flag</A>
+##  is true, then all partitons of size at most <A>k</A> are given.
```

Comment:
The same as for the GAPDoc comment applies here. but honestly, I would not bother updating this duplicate documentation. Instead, I would suggest to simply remove most of this comment (i.e., leave it up to and including line 2452, and remove the rest).

Reply:
Ok, I removed this part of the comments